frontend/src/libs/components/backward-button/back-button-wrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import { BackButton } from './components/backward-button.js';
+
+type BackButtonWrapperProperties = {
+  onGoBack: () => void;
+  isVisible?: boolean;
+};
+
+const BackButtonWrapper: React.FC<BackButtonWrapperProperties> = ({
+  isVisible = false,
+  onGoBack,
+}) => {
+  return (
+    <div className={getValidClassNames(!isVisible && 'visually-hidden')}>
```

Comment:
Cannot we add 'visually-hidden' for <div> wrapper over BackButtonWrapper

Reply:
No, we can't. As here we have div (container) and has its own functionality. Because in meditation list we don't need to have this back button in desktop, but the container is needed. So I can't move this logic to it. That's why I created this BackButtonWrapper. 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/102380564/f93df7bf-7f01-4478-8199-b7019356b938)

Reply:
I can delete this component and just do the same with the usual div there, but as I need this in several components I decided to make it as a component to reduce the repetition.

Reply:
Ш mean to wrap BackButtonWrapper with separate <div></div>

Reply:
I don't understand why we need one extra div here, as I have it in this BackButtonWrapper , which I created exactly for this purpose. Could you explain this, please?

Reply:
Sorry, I missed that, let's proceed with your solution